src/components/RichTextEditor/serializer/markdown/utils.ts

```diff
@@ -2,6 +2,11 @@
 
 import { OptionType, PartialOptionType, defaultNodeTypes } from './types';
 
+const MENTION_ID_REGEX = '[\\d=a-z]+';
+const MENTION_TYPE_REGEX = '[a-z]+';
+export const MENTION_SERIALIZE_REGEX = new RegExp(`@\\[${MENTION_TYPE_REGEX}:${MENTION_ID_REGEX}\\]`, 'gi');
```

Comment:
Case insensitive necessary?
```suggestion
export const MENTION_SERIALIZE_REGEX = new RegExp(`@\\[${MENTION_TYPE_REGEX}:${MENTION_ID_REGEX}\\]`, 'g');
```

Reply:
yes because the ID has small and capitalize letters.